.github/workflows/deploy.yml

```diff
@@ -11,21 +11,28 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      
-      # Use GitHub Actions' cache to shorten build times and decrease load on servers
-      - uses: actions/cache@v2

```

Comment:
Why did u remove this?

Reply:
This actions caches the `vendor/bundle`. We are using ruby plugin which already has bundler cache.

Reply:
Okay, not sure how it works. If it doesn't work, we can circle back to it later.